src/ColinHDev/CPlot/utils/ParseUtils.php

```diff
@@ -109,7 +109,7 @@ private static function parseBlockFromCompressedTreeRoot(string $compressedTreeR
             return null;
         }
         try {
-            $blockState = BlockStateData::fromNbt($compoundTag);
+            $blockState = GlobalBlockStateHandlers::getUpgrader()->upgradeBlockStateNbt($compoundTag);
         } catch (BlockStateDeserializeException) {
             return null;
         }
```

Comment:
```suggestion
        try {
            $blockState = BlockStateData::fromNbt($compoundTag);
        } catch (BlockStateDeserializeException) {
            return null;
        }
        $blockState = GlobalBlockStateHandlers::getUpgrader()->getBlockStateUpgrader()->upgrade($blockState);
```
Might be better. 
The only difference between this way and `upgradeBlockStateNbt()` would be that `upgradeBlockStateNbt()` also checks if the compound tag is a pre-1.13 block state. And if I am not wrong, this should never be the case since CPlot deals with blocks consisting of ids and metas differently in other places. So by using `upgrade()` we can skip that step directly.